src/main/java/chess/view/OutputView.java

```diff
@@ -0,0 +1,57 @@
+package chess.view;
+
+import static chess.view.GameExecutionCommand.END;
+import static chess.view.GameExecutionCommand.MOVE;
+import static chess.view.GameExecutionCommand.START;
+
+import chess.domain.board.ChessBoard;
+import chess.domain.piece.Piece;
+import chess.domain.position.ColumnPosition;
+import chess.domain.position.Position;
+import chess.domain.position.RowPosition;
+import java.util.StringJoiner;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class OutputView {
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final String POSITION_EMPTY_MESSAGE = ".";
+
+    public void printStartMessage() {
+        System.out.println(resolveStartMessage());
+    }
+
+    public void printChessBoardMessage(ChessBoard chessBoard) {
+        System.out.println(resolveChessBoardMessage(chessBoard));
+    }
+
+    private String resolveStartMessage() {
+        return new StringJoiner(LINE_SEPARATOR)
+                .add("체스 게임을 시작합니다.")
+                .add(String.format("> 게임 시작 : %s", START.getCode()))
+                .add(String.format("> 게임 종료 : %s", END.getCode()))
+                .add(String.format("> 게임 이동 : %s source위치 target위치 - 예. %s b2 b3", MOVE.getCode(), MOVE.getCode()))
+                .toString();
+    }
```

Comment:
이건 취향 차이지만, 내가 쓰는 방법을 추가해봤어~

```suggestion
    private String resolveStartMessage() {
        return new StringJoiner(LINE_SEPARATOR)
                .add("체스 게임을 시작합니다.")
                .add("> 게임 시작 : %s".formatted(START.getCode()))
                .add("> 게임 종료 : %s".formatted(END.getCode())))
                .add("> 게임 이동 : %s source위치 target위치 - 예. %s b2 b3".formatted(MOVE.getCode(), MOVE.getCode())))
                .toString();
    }
```